vars/buildPackage.groovy

```diff
@@ -136,57 +143,105 @@ def call(description=null, pkgList=null, buildCmd=null, buildArm=false, changesP
                             currentBuild.result = 'SUCCESS'
                         }
-
+                        try {
+                            unstash 'source-dsc'
+                            unstash 'source-tar'
+                        } catch (e) {
+                            print "Unstash failed, ignoring - no source packages"
+                            currentBuild.result = 'SUCCESS'
+                        }
                         if (isCustomBuild()) {
                             echo "Build not started from official Git repository! Artifacts are not uploaded to external repository"
                             return
                         }
 
-                        files = findFiles(glob: '**/*.deb')
-                        if (files) {
-                            echo "Uploading Artifacts to external repository"
-                            copyArtifacts fingerprintArtifacts: true, projectName: '${JOB_NAME}', selector: specific('${BUILD_NUMBER}')
-
-                            // build up some fancy groovy variables so we do not need to write/copy
-                            // every option over and over again!
-                            def RELEASE = getGitBranchName()
-                            if (getGitBranchName() == "master")
-                                RELEASE = 'current'
-
-                            def VYOS_REPO_PATH = '/home/sentrium/web/dev.packages.vyos.net/public_html/repositories/' + RELEASE
-                            if (getGitBranchName() == "crux")
-                                VYOS_REPO_PATH += '/vyos'
-
-                            def SSH_OPTS = '-o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o LogLevel=ERROR'
-                            def SSH_REMOTE = env.DEV_PACKAGES_VYOS_NET_HOST // defined as global variable
-                            def SSH_DIR = '~/VyOS/' + RELEASE
-
-                            // publish build result, using SSH-dev.packages.vyos.net Jenkins Credentials
-                            sshagent(['SSH-dev.packages.vyos.net']) {
-                                sh(script: "ssh ${SSH_OPTS} ${SSH_REMOTE} -t \"bash --login -c 'mkdir -p ${SSH_DIR}'\"")
-                                echo "Uploading package(s) and updating package(s) in the repository ..."
+                        echo "Uploading Artifacts to external repository"
+                        copyArtifacts fingerprintArtifacts: true, projectName: '${JOB_NAME}', selector: specific('${BUILD_NUMBER}')
+
+                        // build up some fancy groovy variables so we do not need to write/copy
+                        // every option over and over again!
+                        def RELEASE = getGitBranchName()
+                        if (getGitBranchName() == "master")
+                            RELEASE = 'current'
+
+                        def VYOS_REPO_PATH = '/home/sentrium/web/dev.packages.vyos.net/public_html/repositories/' + RELEASE
```

Comment:
Do you think we could move these paths to the environment variable definitions rather than hardcode them?